File: Udemy/Algoritmos e Logica de Programacao/C/Estrutura Repetitiva/ProblemaSequenciaImpares.c
/*
Problema "sequencia_impares" (adaptado de URI 1067)
Leia um valor inteiro X. Em seguida mostre os ímpares de 1 até X, um valor por linha, inclusive o X,
se for o caso.
*/

#include<stdio.h>
#include<stdlib.h>
int main (){
	
	int x, i;

	printf("Digite o valor de X: ");
	scanf("%d",&x);
	
	for(i=0;i<x;i++){
		if(i%2!=0){
			printf("%d\n",i);
		}
	}

    return 0;
}

File: Udemy/Algoritmos e Logica de Programacao/Python/Estrutura Repetitiva/ProblemaSequenciaImpares.py
'''Problema "sequencia_impares" (adaptado de URI 1067)
Leia um valor inteiro X. Em seguida mostre os ímpares de 1 até X, um valor por linha, inclusive o X,
se for o caso.'''

x: int

x = int(input("Digite o valor de X: "))

for i in range (1,x+1):
    if i%2!=0:
        print(i)